probe-rs/src/session.rs

```diff
@@ -832,8 +832,10 @@ fn get_target_from_selector(
             let (returned_probe, found_target) = crate::vendor::auto_determine_target(probe)?;
             probe = returned_probe;
 
-            // Now we can deassert reset in case we asserted it before. This is always okay.
-            probe.target_reset_deassert()?;
```

Comment:
Why is this a problem? Not unhappy with the change, just trying to understand.

Reply:
some probes (ftdi) don't implement it (or assert), so it errors

Reply:
Ah, shoot, good catch, thanks!